packages/web-react/src/hooks/useValidationText.tsx

```diff
@@ -1,30 +1,44 @@
-import React, { ReactNode, useMemo } from 'react';
+import React, { ElementType, ReactNode, useMemo } from 'react';
 import { ValidationState, ValidationTextType } from '../types';
 
 export interface UseValidationTextProps {
   validationTextClassName?: string;
   validationState?: ValidationState;
   validationText?: ValidationTextType;
+  validationElementType?: ElementType;
+  /** @deprecated Will be removed in the next major version. */
+  requireValidationState?: boolean;
 }
 
 export const useValidationText = (props: UseValidationTextProps): ReactNode => {
-  const { validationTextClassName, validationState, validationText } = props;
+  const {
+    validationTextClassName,
+    validationState,
+    validationText,
+    validationElementType: ElementTag = 'div',
+    /** @deprecated Will be removed in the next major version. */
+    requireValidationState = true,
+  } = props;
 
   return useMemo(() => {
-    if (!(validationState && validationText)) {
+    if (requireValidationState && !validationState) {
       return;
     }
 
-    const getMessage = (message: string, index?: number) => (
-      <div className={validationTextClassName} key={`validationText__${index}`}>
-        {message}
-      </div>
-    );
+    if (!validationText) {
```

Comment:
This can be merged with the previous condition.

Reply:
You fix this in your PR.